examples/server_client.rs

```diff
@@ -0,0 +1,100 @@
+//! Note that the terms "client" and "server" here are purely what we logically associate with them.
+//! Technically, they both work the same.
+
+extern crate laminar;
+
+use std::{
+    io::stdin,
+    net::{SocketAddr, ToSocketAddrs},
+};
+
+use laminar::{
+    error::Result,
+    Packet,
+    UdpSocket,
+};
+
+const SERVER: &str = "localhost:12351";
+
+fn addr(s: &str) -> Result<SocketAddr> {
```

Comment:
Remove this and replace it with  `let addr: SocketAddr = string.parse().unwrap();` See comment below.